ext/readme.md

```diff
@@ -49,7 +49,7 @@ ext | `pysrt` | [1.1.0](https://pypi.org/project/pysrt/1.1.0/) | `subliminal` |
 ext | `python-dateutil` | [2.8.1](https://pypi.org/project/python-dateutil/2.8.1/) | **`medusa`**, `guessit`, `imdbpie`, `tvdbapiv2` | Module: `dateutil`
 ext | `python-twitter` | [3.5](https://pypi.org/project/python-twitter/3.5/) | **`medusa`** | Module: `twitter`
 ext | `pytz` | [2018.4](https://pypi.org/project/pytz/2018.4/) | `subliminal` | -
-**ext2 ext3** | `PyYAML` | [5.1](https://pypi.org/project/PyYAML/5.1/) | `knowit` | Module: `yaml`
+**ext2 ext3** | **`PyYAML`**<br>`yaml` | [5.3.1](https://pypi.org/project/PyYAML/5.3.1/) | `knowit` | Module: `_yaml`
```

Comment:
This will fix itself the next time someone updates.
@medariox You had `ext2/_yaml` or `ext3/_yaml` folder from the last time you updated directly using `pip`, and `mvt` picked that up by mistake.